packages/client/src/schema-component/antd/page/Page.tsx

```
@@ -144,13 +136,23 @@ const PageHeader = (props) => {
+              {Object.values(extendComponents)?.map((item: any, index) => {
+                const schema = {
+                  type: 'void',
+                  name: item.name,
+                  'x-component': item.component,
+                  'x-component-props': {},
+                } as ISchema;
+                const componentProps = { ...props, isExtra: true, fieldSchema };
+                return (
+                  <PageExtendComponentProvider {...componentProps}>
+                    <SchemaComponent schema={schema} />
+                  </PageExtendComponentProvider>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)